angular/src/types/mezon/webview/types.d.ts

```
@@ -0,0 +1,31 @@
+  IframeWillReloaded = "iframe_will_reload",
+}
+export type MezonEventHandler<T> = (eventType: MezonAppEvent, eventData?: T) => void;
+export type EventHandlers<T> = Record<string, MezonEventHandler<T>[]>;
+export type InitParams = Record<string, string | null>;
+export interface IMezonWebView {
+  initParams: InitParams;
+  isIframe: boolean;
+  onEvent<T>(eventType: MezonAppEvent, callback: MezonEventHandler<T>): void;
+  offEvent<T>(eventType: MezonAppEvent, callback: MezonEventHandler<T>): void;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+  postEvent<T>(eventType: MezonWebViewEvent, eventData: T, callback: Function): void;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)